x-pack/plugins/apm/public/components/app/Main/UpdateBreadcrumbs.ts

```diff
@@ -5,18 +5,23 @@
  */
 
 import { Location } from 'history';
-import { flatten } from 'lodash';
+import { flatten, last } from 'lodash';
 import React from 'react';
 // @ts-ignore
 import { withBreadcrumbs } from 'react-router-breadcrumbs-hoc';
 import chrome from 'ui/chrome';
 import { toQuery } from '../../shared/Links/url_helpers';
-import { routes } from './routeConfig';
+import { BreadcrumbFunction, BreadcrumbProps, routes } from './routeConfig';
+
+interface BreadcrumbElement {
+  type: BreadcrumbFunction;
+  props: BreadcrumbProps;
+}
```

Comment:
This seems like something we should be getting from React or `react-router-breadcrumbs-hoc`. 
`react-router-breadcrumbs-hoc` now ships with Typescript support. Would you mind updating as part of this UI, and see if they have types we can use?